### PR TITLE
Fix Nonsense

### PR DESCRIPTION
**Pull request by AntiCheap**
Removed dead code and straight up madness, there is a lot to be improved.

### CODE REVIEW

**Review by KozmikNano — APPROVED** [review]
Looks good

File: main.js [hunk 1]
```diff
@@ -454,21 +437,21 @@ function userInput(cmd, DiscordOrigin, discordMsg, channel) {
 			console.log("Syntax: status, enable, disable");
 			break;
 		case "loop status":
-			if (JSON.stringify(webserver.restartQueue) == "true")
```
Comment (themoonisacheese) [inline main.js:457]: this has the be the funniest code i read this week lmao

**Comment by nickcat325** [comment]
@AntiCheap You cleaned up some of my code. Thanks.

**Comment by mutexdeveloper** [comment]
holy shit 🤣

**Pull request merged**